### PR TITLE
Updated time units and rotations specification in create header

### LINKED ISSUE #7: Change floats to integers
"Max Time" is currently specified in floats but should probably rather be integer in usec of msec.

### PR DESCRIPTION
**Pull request by AckslD**
closes #7

### CODE REVIEW

File: draft-dahlberg-ll-quantum-03.xml [hunk 1]
```diff
@@ -280,28 +283,47 @@ is to manage re-tries in producing entanglement at the the physical layer.
...
-             </t>
+            </t>
+             <t> Probability distributions used to sample random basis for measure directly:
+                 <list style="symbols"> 
+                   <t>(PL1) (8 bits): Parameter for local probability distribution used to sample basis if RL is not 00</t>
+                   <t>(PL2) (8 bits): Parameter for local probability distribution used to sample basis if RL is not 00</t>
+                   <t>(PR1) (8 bits): Parameter for remote probability distribution used to sample basis if RR is not 00</t>
+                   <t>(PR2) (8 bits): Parameter for remote probability distribution used to sample basis if RR is not 00</t>
+                 </list>
+                 Each value is seen as the integet representing of the binary value.
```
Comment (mdskrzypczyk) [inline draft-dahlberg-ll-quantum-03.xml]: integer*

File: draft-dahlberg-ll-quantum-03.xml [hunk 2]
```diff
@@ -254,8 +254,11 @@ is to manage re-tries in producing entanglement at the the physical layer.
                A binary value encoding the integer 'n' represents the fidelity 'n' divided by (2^16-1).
              </t>
              <t>
-               Max Time (16 bits): The maximum time in milliseconds that the higher layer is willing to wait for the request to be fulfilled.
-               Represented as a binary16 float as specified in <xref target="binary16"/>.
+               Time unit (tu) (2 bits): The time units used for specifying Max Time, where (00, 01, 10) each indicate (micro-seconds, milliseconds, seconds) respectively and 11 is unused.
+             </t>
+             <t>
+               Max Time (16 bits): The maximum time in the time units specified above that the higher layer is willing to wait for the request to be fulfilled.
```
Comment (mdskrzypczyk) [inline draft-dahlberg-ll-quantum-03.xml]: Max Time and Time unit together make up 16 bits, so Max Time here needs to be 14 bits and the header diagram also needs to be updated to make sure Max Time is split into Max Time and tu

**Review by mdskrzypczyk — CHANGES_REQUESTED** [review]
Two small things

File: draft-dahlberg-ll-quantum-03.xml (hunk 1 above)
Comment (AckslD) [inline draft-dahlberg-ll-quantum-03.xml]:
```suggestion
                 Each value is seen as the integer representing of the binary value.
```

File: draft-dahlberg-ll-quantum-03.xml (hunk 2 above)
Comment (AckslD) [inline draft-dahlberg-ll-quantum-03.xml]:
```suggestion
               Max Time (14 bits): The maximum time in the time units specified above that the higher layer is willing to wait for the request to be fulfilled.
```

File: draft-dahlberg-ll-quantum-03.xml (hunk 1 above)
Comment (AckslD) [reply draft-dahlberg-ll-quantum-03.xml]: Done

File: draft-dahlberg-ll-quantum-03.xml (hunk 2 above)
Comment (AckslD) [reply draft-dahlberg-ll-quantum-03.xml]: Done

**Pull request merged**